nixos/tests/all-tests.nix

```diff
@@ -230,6 +230,7 @@ in {
   couchdb = handleTest ./couchdb.nix {};
   crabfit = handleTest ./crabfit.nix {};
   cri-o = handleTestOn ["aarch64-linux" "x86_64-linux"] ./cri-o.nix {};
+  criterion = handleTestOn ["aarch64-linux" "x86_64-linux"] ./criterion/default.nix {};
```

Comment:
The NixOS testing infrastructure isn't meant for these kinds of tests. Try using `checkPhase` within the package itself instead.

Reply:
wouldn't setting the `checkPhase` would override the tests done by meson?